packages/system/src/toCSS.ts

```diff
@@ -1,4 +1,61 @@
-export const toCssUnit = (token: string | number) => {
-  if (typeof token === "string") return token;
-  else return `${token}px`;
+/**
+ * An object of css properties that don't accept values with units
+ * @link https://github.com/emotion-js/emotion/tree/main/packages/unitless
+ */
+const unitless = {
```

Comment:
I know this is from emotion, but why is this an object instead of a list of keys?

Reply:
O(1) lookup?

Reply:
Make sense!